vantage6/server/configuration/configuration_manager.py

```diff
@@ -1,3 +1,6 @@
+# FIXME BvB 2021-12-21
+# I think this is deprecated by vantage.cli.configuration_manager. Remove?!
+
```

Comment:
You mean the entire validation? I remember it being quite annoying at times 😨.

Reply:
I meant that the entire file should probably be removed as I don't think its code is still being used